doc/5_saving.md

```diff
@@ -77,6 +77,70 @@ val collection = sc.parallelize(Seq(WordCount("dog", 50), WordCount("cow", 60)))
 collection.saveToCassandra("test", "words2", SomeColumns("word", "num" as "count"))
 ```
 
+## Modifying CQL Collections
+The default behavior of the Spark Cassandra Connector is to overwrite collections when inserted into
+a cassandra table. To override this behavior you can specify a custom mapper with instructions on
+how you would like the collection to be treated.
+
+The following operations are supported
+
+- append/add  (lists, sets, maps)
+- prepend (lists)
+- remove  (lists, sets)
+- overwrite (lists, sets, maps) :: Default
+
+Remove is not supported for Maps.
+
+These are applied by adding the desired behavior to the ColumnSelector
+
+Example Usage
+
+Takes the elements from rddSetField and removes them from corrosponding C* column
+"a_set" and takes elements from "rddMapField" and adds them to C* column "a_map" where C*
+column key == key in the RDD elements. 
+   
+    ("key", "a_set" as "rddSetField" remove , "a_map" as "rddMapField" as append)
```

Comment:
should I put "as" before remove or not? There is as before append, but not before remove.